Makefile.rules

```diff
@@ -34,9 +34,13 @@ ifeq ($(NO_CXNG),)
 INCLUDES_PATH += $(BOLOS_SDK)/lib_cxng/include
 endif
 
-SOURCE_PATH   += $(BOLOS_SDK)/src $(foreach libdir, $(SDK_SOURCE_PATH), $(dir $(shell find $(BOLOS_SDK)/$(libdir) -name '*.[csS]'))) $(dir $(shell find $(APP_SOURCE_PATH) -name '*.[csS]'))
+ifeq ($(USE_COMMON_APP_FILES_IN_SDK), 1)
+COMMON_APP += $(BOLOS_SDK)/common_app
```

Comment:
I'm wondering if it would be better to name the folder something like `lib_app_helper` to be similar to `lib_bagl`, `lib_ux`, etc..
And also to enable it with a more classical way, that would be adding it in `SDK_SOURCE_PATH` or  `COMMON_SDK_SOURCE_PATH`.
what do you think of this?